web/frontend/src/types/frontendRequestBody.ts

```diff
@@ -1,5 +1,13 @@
 import { ID } from './configuration';
 
+const INITIAL_STATUS = 0;
```

Comment:
Just a question but wouldn't it be easier to have an actual const enum Status ?
`const enum Status {
  Initial,
  Open,
  Closed,
  Shuffled,
  Decrypted,
  ResultAvailable,
  Canceled
}`

Reply:
Yes good point !